NetKAN/AtmosphereAutopilot.netkan

```diff
@@ -1,11 +1,11 @@
 {
-    "spec_version" : "v1.4",
+    "spec_version" : "v1.5.2",
```

Comment:
`spec-version` doesn't change. `spec_version` is the version of the CKAN specification that a metadata file is using (for example if you're using the `find` function you need `"v1.4"`.